src/composables/useTour.ts

```diff
@@ -0,0 +1,393 @@
+                    '.account-overview .backup-warning',
+                    '.account-overview .mobile-menu-bar',
+                    '.account-overview .account-balance-container',
+                    '.account-overview .address-list',
+                    '.account-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.account-overview .bitcoin-account',
+                ],
+            },
+        },
+        [MobileOnboardingTourStep.WALLET_BALANCE]: {
+            path: '/',
+            tooltip: {
+                target: '.account-overview .account-balance-container .amount',
+                content: [
+                    'Check the bar-chart to see how your addresses compose your total balance.',
+                    'Currently you have 100% NIM, and no BTC.',
+                ],
+                params: {
+                    placement: 'bottom',
+                },
+            },
+            ui: {
+                elementsWithOpacity: [
+                    '.account-overview .backup-warning',
+                    '.account-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.account-overview .mobile-menu-bar',
+                    '.account-overview .account-balance-container',
+                    '.account-overview .address-list',
+                    '.account-overview .bitcoin-account',
+                ],
+            },
+        },
+        [MobileOnboardingTourStep.RECOVERY_WORDS_ALERT]: {
+            path: '/',
+            tooltip: {
+                target: '.account-overview .backup-warning button',
+                content: [
+                    'There is no \'forgot password\'.',
+                    'Create a backup to make sure you stay in control.',
+                ],
+                params: {
+                    placement: 'bottom',
+                },
+            },
+            ui: {
+                elementsWithOpacity: [
+                    '.account-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.account-overview .account-balance-container',
+                    '.account-overview .address-list',
+                    '.account-overview .bitcoin-account',
+                ],
+            },
+        },
+        [MobileOnboardingTourStep.MENU_ICON]: {
+            path: '/',
+            tooltip: {
+                target: '.account-overview .mobile-menu-bar > button.reset',
+                content: [
+                    'Tap on the menu icon to access your account and wallet settings.',
+                ],
+                params: {
+                    placement: 'bottom-start',
+                },
+            },
+            lifecycle: {
+                prepareDOMNextPage: async () => {
+                    await openAccountOptionsModal();
+                },
```

Comment:
What you are doing here, I believe doesn't scale well, e.g. when random steps may be skipped. Instead of having the previous and following steps be responsible for preparing the state for the next step, the step itself should be in charge of preparing the DOM as it needs it. So basically every step should have a `mounted` and `unmounted` callback (or called `loaded` and `unloaded`, you get the idea), in which (for this example) the account menu is opened and closed. Not the previous step be responsible for opening it.

Reply:
Yes, this makes a tone of sense. I will change it to `mounted` and `unmounted`